sql/core/src/main/scala/org/apache/spark/sql/execution/adaptive/QueryStageExec.scala

```diff
@@ -161,10 +161,14 @@ case class ShuffleQueryStageExec(
       case _ =>
     }
   }
+}
 
-  def mapStats: MapOutputStatistics = {
```

Comment:
Can we make this return `Option[MapOutputStatistics]` and document it? It should be easier to use than an exactor.

Reply:
updated.